### ISSUE TITLE
Request: min_interval to Range Slider

### ISSUE THREAD

**Issue by Nosferican** [rstudio/shiny#866]
The min_interval option from the ion.rangeSlider is not incorporated to the sliderInput().

References:
https://github.com/IonDen/ion.rangeSlider
http://shiny.rstudio.com/reference/shiny/latest/sliderInput.html

**Comment by daattali** [comment]
Better yet, maybe the slider range input could support an arbitrary list of options that it would just pass along to ionRangeSlider?  I think that's what selectize inputs do.  Perhaps there's a good reason not to do that, but otherwise, it seems like a better solution because there are some other options that are also not implemented that could be useful

**Comment by Nosferican** [comment]
Indeed. If possible to make it play nice and adapt the full functionality of the ionRangeSlider that could be pretty useful and improve this widget.

**Comment by alexwhan** [comment]
Another call for support of all the ionRangeslider parameters, would be really useful for a range of applications in data interaction

**Comment by alandipert** [comment]
In previous design discussions we (the Shiny team) have wavered in our desire for enhancements like this. Our historical argument against tighter coupling with ionRangeslider is something like:  the more ionRangeslider-specific options we expose, the more dependent on a particular version of ionRangeslider we become. With more dependency comes more vulnerability to downstream breaking changes, and so degrades the stability of Shiny.

As @daattali points out, our relationship with ionRangeslider is very similar to that with Selectize.

In any case, it's high time for us to revisit this discussion, and to consider seriously ionRangeslider enhancements. It looks like ionRangeslider development has been sporadic these past few years, which is a data point against the stability argument.

![Screen Shot 2019-04-26 at 1 11 47 PM](https://user-images.githubusercontent.com/26024/56833956-f4ff6380-6824-11e9-88cd-1eea1a19f61b.png)
